doc/man3/SSL_CIPHER_get_name.pod

```diff
@@ -91,6 +93,15 @@ TLS 1.3 cipher suites) B<NID_auth_any> is returned. Examples (not comprehensive)
  NID_auth_ecdsa
  NID_auth_psk
 
+SSL_CIPHER_get_prf_nid() retuns the pseudo-random function NID for B<c>. If B<c> is
```

Comment:
Should this get submitted as a separate bugfix PR to openssl?

Reply:
Probably